test/detectLocalGit.js

```diff
@@ -57,8 +57,8 @@ function _cleanTempGitDir() {
 
 function _deleteFolderRecursive(dir) {
 
-  if (!dir.match('node-coveralls/test')) {
-    throw new Error('Tried to clean a temp git directory that did not match path: node-coveralls/test');
+  if (!dir.includes(path.normalize('node-coveralls/test'))) {
```

Comment:
Switch to `includes` because on Windows the path contains backslashes which need to be escaped. So is simpler IMO.

Reply:
I need to confirm this. If it's supposed to be the same dir, we can do just an equality check. Unfortunately, I can't do much until #235 is merged since I'm on Windows :/

Reply:
Actually, nvm, the change is correct. `dir` is an absolute path.